<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Buttons</class>
 <widget class="QWidget" name="Buttons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(29,63,67);</string>
  </property>
  <widget class="QPushButton" name="ButCalibration">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>610</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/CALIBRATIONIMG.png);
background-color:transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ButDoctor">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>610</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/DOCTORIMG.png);
background-color:transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ButMeasure">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>610</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/MEASUREIMG.png);
background-color:transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ButCurrent">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>10</y>
     <width>41</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/3dot.png);
background-color:transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ButSettings">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>610</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/SETTINGSIMG.png);
background-color:transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ButLens">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>610</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/LENSIMG.png);
background-color:transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-30</y>
     <width>1291</width>
     <height>661</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QPushButton" name="ButPatTouch">
     <property name="geometry">
      <rect>
       <x>1180</x>
       <y>440</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/image/assesstive.png);
background-color:transparent;
border:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButPatAdd">
     <property name="geometry">
      <rect>
       <x>1140</x>
       <y>350</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
image: url(:/image/addperson.png);
background-color:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButPatDelete">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>410</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
image: url(:/image/delete.png);
background-color:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButPatEdit">
     <property name="geometry">
      <rect>
       <x>1090</x>
       <y>490</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color:transparent;
image: url(:/image/edit.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButPatView">
     <property name="geometry">
      <rect>
       <x>1160</x>
       <y>540</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color:transparent;
image: url(:/image/viewbut.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>861</width>
       <height>531</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(30, 131, 140);</string>
     </property>
    </widget>
    <zorder>tableView</zorder>
    <zorder>ButPatEdit</zorder>
    <zorder>ButPatTouch</zorder>
    <zorder>ButPatView</zorder>
    <zorder>ButPatDelete</zorder>
    <zorder>ButPatAdd</zorder>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>531</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(30, 131, 140);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButDocEdit">
     <property name="geometry">
      <rect>
       <x>1090</x>
       <y>480</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color:transparent;
image: url(:/image/edit.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButDocTouch">
     <property name="geometry">
      <rect>
       <x>1190</x>
       <y>430</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/image/assesstive.png);
background-color:transparent;
border:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButDocView">
     <property name="geometry">
      <rect>
       <x>1180</x>
       <y>540</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color:transparent;
image: url(:/image/viewbut.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButDocDelete">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>380</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
image: url(:/image/delete.png);
background-color:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButDocAdd">
     <property name="geometry">
      <rect>
       <x>1170</x>
       <y>320</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/image/adddoctor.png);
border:none;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QTableView" name="tableView_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(30, 131, 140);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButLensEdit">
     <property name="geometry">
      <rect>
       <x>1180</x>
       <y>530</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color:transparent;
image: url(:/image/edit.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButLensTouch">
     <property name="geometry">
      <rect>
       <x>1190</x>
       <y>430</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/image/assesstive.png);
background-color:transparent;
border:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButLensDelete">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>440</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
image: url(:/image/delete.png);
background-color:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ButLensAdd">
     <property name="geometry">
      <rect>
       <x>1170</x>
       <y>330</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color:transparent;
image: url(:/image/addlens.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>240</y>
       <width>391</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(118, 223, 250);
font: 75 36pt &quot;Ubuntu&quot;;
font:bold;</string>
     </property>
     <property name="text">
      <string>Measure</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>250</y>
       <width>391</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(118, 223, 250);
font: 75 36pt &quot;Ubuntu&quot;;
font:bold;</string>
     </property>
     <property name="text">
      <string>Calculator</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_8">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>200</y>
       <width>391</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(118, 223, 250);
font: 75 36pt &quot;Ubuntu&quot;;
font:bold;</string>
     </property>
     <property name="text">
      <string>Calibration</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>220</y>
       <width>391</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(118, 223, 250);
font: 75 36pt &quot;Ubuntu&quot;;
font:bold;</string>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="ButPatient">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>610</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/PATIENTIMG.png);
background-color:transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ButCalculator">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>610</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/CALCULATORIMG.png);
background-color:transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>tabWidget</zorder>
  <zorder>ButPatient</zorder>
  <zorder>ButDoctor</zorder>
  <zorder>ButLens</zorder>
  <zorder>ButMeasure</zorder>
  <zorder>ButCalculator</zorder>
  <zorder>ButCalibration</zorder>
  <zorder>ButSettings</zorder>
  <zorder>ButCurrent</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
